Students/Benny/3. Protocols and bridging/Questions.md

```diff
@@ -32,3 +32,9 @@
 3. Explain the difference between NSOperation and GDC please.
 
 4. NSDictionary 'bindings' argument in NSPredicate filtering?
+
+## SideNote
+
+1. When having to react to delegates, like for example UITextFieldDidEndEditing(), it seems unlikely that you're not going to retain your viewModel inside the collectionViewCell. Isn't "never" a bit harsh?
```

Comment:
As debated this is not the way to go. I know we do it but it brakes design pattern.

Always delegate changes to the ViewController so he can handle the changes:

1. Deside if screen needs to be updated
2. Save changes if needed
3. Update the ViewModel from the detail changes.